app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = []
+    for person_data in people:
+        Person.people[person_data["name"]]\
+            = Person(person_data["name"], person_data["age"])
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people.get(person_data["wife"])
```

Comment:
You could simplify the condition by using get and walrus operator
```suggestion
        if wife_name := person_data.get("wife"):
            person.wife = Person.people.get(wife_name)
```
Apply this change to the husband condition as well